src/components/Link/SbLink.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <component
+    :is="as"
+    class="sb-link"
+    :class="computedClasses"
+    :title="title"
+    :href="href"
+    :to="to"
+    v-on="$listeners"
+  >
+    <SbIcon v-if="icon" :name="icon" />
+    <span v-if="label" class="sb-link__label">{{ label }}</span>
+    <span v-else-if="$slots.default" class="sb-link__label"><slot /></span>
```

Comment:
It seems we could break more lines here to make the code more readable

```
<span v-if="label" class="sb-link__label">
  {{ label }}
</span>
<span v-else-if="$slots.default" class="sb-link__label">
   <slot />
</span>
```

Reply:
I agree with Pablo here